### PR TITLE
release 10.12.0 rc2

### PR DESCRIPTION
**Pull request by jnweiger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ ownCloud is an open-source file sync, share and content collaboration software t
 
 ## Docker Tags and respective Dockerfile links
 
+- [`10.12.0-rc2`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.12.0-rc2`
```
Comment (xoxys) [inline README.md:34]: As always, please don't add RC releases to the README.

File: .drone.star [hunk 2]
```diff
@@ -1,5 +1,14 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.12.0-rc2",
+            "qa": "https://download.owncloud.com/server/testing/owncloud-complete-20230228-qa.tar.bz2",
+            "tarball": "https://download.owncloud.com/server/stable/owncloud-complete-20230228.tar.bz2",
+            "tarball_sha": "86a9895f22be820eef7c7434cadea1b7476c8e5416fec902526d9a626a0c85c4",
+            "php": "7.4",
+            "base": "v20.04",
+            "tags": [],
```
Comment (xoxys) [inline .drone.star:10]: Don't know if there is a fallback to `value` but I would recommend setting the tags list to `"tags": ["10.12.0-rc2"]`.

**Review by xoxys — DISMISSED** [review]

File: .drone.star (hunk 2 above)
Comment (jnweiger) [reply .drone.star:10]: Same here. The instruction says empyt tags for RC. I'll change that too. Thanks for the hints.

File: README.md (hunk 1 above)
Comment (jnweiger) [reply README.md:34]: ok. the instruction says 'all' to te README. I'll change that.

**Comment by xoxys** [comment]
CI is failing because `mailhog` was replaced by `inbucket` in core tests. Need to find a way to handle this for oC 10.10 - 10.12 consistently. Will work on this later, if you need the RC build urgently let me know.

**Comment by xoxys** [comment]
It's not that simple and will fail for the older 10.11 releases now.

**Comment by jnweiger** [comment]
@xoxys @phil-davis CI is green, please approve fast, before it changes its mind again!

**Comment by jnweiger** [comment]
> Please dont merge it in that state... As mentioned the conditioning stuff is not neccessary and I would like to use the daily qa bundle by default to avoid it.

Ok. In that case I need some help. We need a working docker image. I had hoped to get it done today.

**Comment by xoxys** [comment]
Thats why I asked hours ago if it is urgend. Merge it, will rework it later.

**Comment by jnweiger** [comment]
Ok. thanks  - I did not see your comment in time. Sorry.

**Review by xoxys — APPROVED** [review]

**Pull request merged**